client/account-recovery/lost-password/test/index.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import reactTapEventPlugin from 'react-tap-event-plugin';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { LostPassword } from '..';
+
+describe( 'LostPassword', () => {
+	it( 'should render as expected', () => {
+		const wrapper = shallow( <LostPassword className="test__test" translate={ noop } /> );
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/8955/commits/d8a935ca34fc623e1a449957df7cb858a65ea79a